src/scales/timeScale.ts

```diff
@@ -47,6 +46,38 @@ export module Scales {
       var startTimeValue = endTimeValue - Plottable.MILLISECONDS_IN_ONE_DAY;
       return [new Date(startTimeValue), new Date(endTimeValue)];
     }
+
+    public scale(value: Date): number {
+      return this._d3Scale(value);
+    }
+
+    protected _getDomain() {
+      return this._d3Scale.domain();
+    }
+
+    protected _setBackingDomain(values: Date[]) {
```

Comment:
@bluong  What is the difference between `_setDomain()` and `_setBackingDomain()` and why do we need both?